api/python/examples/complete.py

```
@@ -0,0 +1,472 @@
+    #       "address_country": "US",
+    #       "metadata": {"tier": "enterprise"},
+    #   }
+
+    try:
+        customer = client.customers.create_customer(request=request)
+        print(f"  Created: id={customer.id!r}  name={customer.name or 'N/A'}")
+        return customer
+    except FlexpriceError as exc:
+        if exc.status_code == 409:
+            # Customer already exists — look it up by external_id
+            print(f"  409 conflict — fetching existing customer …")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)